spinnaker-config/powerdummy-populator/powerdummy.yaml

```diff
@@ -0,0 +1,21 @@
+schema: "1"
+id: powerdummyPopulator
+pipeline:
+  application: powerdummy
```

Comment:
`powerdummy-populator`. Needs to match the subfolder name.

Reply:
fixed